clicommand/agent_start.go

```diff
@@ -851,6 +858,7 @@ var AgentStartCommand = cli.Command{
 			PluginValidation:                !cfg.NoPluginValidation,
 			LocalHooksEnabled:               !cfg.NoLocalHooks,
 			AllowedRepositories:             cfg.AllowedRepositories,
+			AllowedPlugins:                  cfg.AllowedPlugins,
```

Comment:
like in the other PR, we should compile these regexes in this file, and then pass them to the `JobRunner` as a `[]*regexp.Regexp`. probably a good idea to backport this behaviour to `allowed-repos` as well

Reply:
Like in the other PR. What other PR?